README.md

```diff
@@ -9,15 +9,17 @@ Directly calling WMBB method in WMI device will lead to a kernel panic. I have n
 ```
 Device (SMCD)
 {
-    Name (_HID, EisaId ("PNP0C02")
+    Name (_HID, EisaId ("PNP0C02"))
     Name (_CID, "MON00000")
-    Mathod (WMIB, 3, Serialized)
+    Method (WMIB, 3, Serialized)
     {
         \_SB.WMI.WMBB(Arg0, Arg1, Arg2)
     }
 }
 ```
 
+put it above `Device (EC)`, but below `Scope (\_SB.PCI0.LPCB)`
```

Comment:
Well, about this one, the device SMCD is a virtual device, here it's only a workaround for kernel panics caused by direct hooking WMI device. So I think it's not necessary to limit its location in DSDT.

Reply:
Alright! I am actually not that very comfortable with DSDT edits, so I thought I would put it before Device (EC). But you are right, it isn't necessary to have a fixed location!